.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+Thanks for submitting an issue! 
+
+Please answer these questions
+
+-------------ANY REQUEST NEEDS THE FOLLOWING --------------------
+
+I. What keywords did you search in Kubernetes issues before filing this one? (If you have found any duplicates, you should instead reply there.):
```

Comment:
I think having an issue (duplicate or not) is more valuable than the risk of potentially losing an issue due to a procedural firewall.. 

Suggestion : 

```
-------------ANY REQUEST NEEDS THE FOLLOWING --------------------

I. Please spend a moment or two looking for duplicate issues. If this happens to be a duplicate, the community will try to point them together.

```